lnclient/lnd/lnd.go

```diff
@@ -241,7 +241,7 @@ func (svc *LNDService) MakeInvoice(ctx context.Context, amount int64, descriptio
 		expiry = lnclient.DEFAULT_INVOICE_EXPIRY
 	}
 
-	resp, err := svc.client.AddInvoice(ctx, &lnrpc.Invoice{ValueMsat: amount, Memo: description, DescriptionHash: descriptionHashBytes, Expiry: expiry})
+	resp, err := svc.client.AddInvoice(ctx, &lnrpc.Invoice{ValueMsat: amount, Memo: description, DescriptionHash: descriptionHashBytes, Expiry: expiry, Private: true})
```

Comment:
I am not sure we should enable this always. LDK only enables it if you have no public channels

Reply:
But what is the guarantee that the public channels connect all the way from sender to us? Aren't we then lessening our chances of a successful payment?

Reply:
How does it work in LDK? Here this also uses public channels after setting the `Private` flag (tested in Polar)